samples/attested_tls/server/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.11)
+
+#find_package(OpenEnclave CONFIG REQUIRED)
```

Comment:
find_package is commented out. If not needed, delete?

Reply:
removed